CHT/CHT.C

```diff
@@ -12,7 +12,7 @@ preciceAdapter::CHT::ConjugateHeatTransfer::ConjugateHeatTransfer
 mesh_(mesh)
 {}
 
-bool preciceAdapter::CHT::ConjugateHeatTransfer::configure(const YAML::Node adapterConfig)
+bool preciceAdapter::CHT::ConjugateHeatTransfer::configure(const IOdictionary adapterConfig)
```

Comment:
why passing by copy instead of const ref?  -- `(const IOdictionary& )`

If you are just getting dictionary values out of it and don't need the IOobject or any registry, then better just to receive as `(const dictionary& dict)`  and that's it.

Reply:
Good point, @olesenm, thank you very much! Passing by const value did not make any sense indeed. I will eventually look for more places in the code where such quick decisions can be revisited and simplify some parts.